src/main/java/uk/gov/justice/hmpps/prison/api/resource/LocationResource.java

```diff
@@ -72,7 +71,6 @@ public ResponseEntity<List<OffenderBooking>> getOffendersAtLocationDescription(
                 .keywords(keywords)
                 .locationPrefix(locationPrefix)
                 .returnAlerts(returnAlerts)
-                .returnIep(returnIep)
```

Comment:
Will remove this flag from `SearchOffenderRequest` in the PR to remove `SearchOffenderResource`